packages/x-components/src/x-modules/facets/store/module.ts

```diff
@@ -47,6 +49,9 @@ export const facetsXStoreModule: FacetsXStoreModule = {
     },
     setFacet(state, facet: Facet) {
       Vue.set(state.facets, facet.id, facet);
+    },
+    setQuery(state, query: string) {
```

Comment:
I think the type here can be omitted, because it is inferred from the `FacetsMutations` interface.

```suggestion
    setQuery(state, query) {
```